test/Form_test.js

```diff
@@ -922,6 +928,30 @@ describe("Form", () => {
           })
         );
       });
+
+      it("should reset errors and errorSchema state to initial state after correction and resubmission", () => {
+        const { comp, node } = createFormComponent({
+          schema,
+          onError: () => {},
+        });
+
+        Simulate.change(node.querySelector("input[type=text]"), {
+          target: { value: "short" },
+        });
+        Simulate.submit(node);
+
+        expect(comp.state.errorSchema).eql({
+          __errors: ["should NOT be shorter than 8 characters"],
+        });
```

Comment:
`+        expect(comp.state.errors.length).eql(1);`

to show that the size has changed.